packages/typespec-ts/src/modular/helpers/classicalOperationHelpers.ts

```diff
@@ -74,21 +74,26 @@ export function getClassicalOperation(
     .filter((i) => i.getName() === interfaceName)[0];
   const properties: OptionalKind<PropertySignatureStructure>[] = [];
   if (layer !== prefixes.length - 1) {
-    properties.push({
-      kind: StructureKind.PropertySignature,
-      name: normalizeName(
-        (layer === prefixes.length - 1
-          ? prefixes[layer]
-          : prefixes[layer + 1]) ?? "",
-        NameType.Property
-      ),
-      type: `${getClassicalLayerPrefix(
-        prefixes,
-        NameType.Interface,
+    const name = normalizeName(
+      (layer === prefixes.length - 1 ? prefixes[layer] : prefixes[layer + 1]) ??
         "",
-        layer + 1
-      )}Operations`
-    });
+      NameType.Property
+    );
+    if (
+      !properties.some((x) => x.name === name) &&
+      !(existInterface && existInterface.getProperty(name))
```

Comment:
I wonder why we need this filter? do you see any cases where there's duplication?

Reply:
Yes, the multipart payload cadl ranch spec causes this duplication for some reason. If you revert this change you should see the issue.